.github/workflows/execute-tests-and-promote.yml

```diff
@@ -38,6 +38,7 @@ jobs:
       - name: "Docker Login"
         uses: docker/login-action@v1
         with:
+          registry: ${{ (!startsWith(secrets.RELEASE_REGISTRY, 'docker.io/')) && secrets.RELEASE_REGISTRY || null }}
```

Comment:
This is worthy of a comment:

- you need to not set the registry if it _does_ start with `docker.io/`
- GHA requires stupid `&&` `||` crap because there's no ternary operator, but really this is a ternary.

Reply:
(It would've been incomprehensible if I hadn't happened to remember the Slack wisecrackery from yesterday. 🙂 )